upgrade/run_upgrade_test.sh

```diff
@@ -47,6 +48,10 @@ function get_git_sha() {
 
   if [[ "${tag}" =~ "latest" ]];then
     release_version=$(echo "${tag}" | cut -d'_' -f1)
+    # shellcheck disable=SC2072
+    if [[ ${release_version} < "1.6" ]]; then
```

Comment:
Note that this is a string comparison not arithmetic comparison. Albeit this should be fine for comparing against a hardcoded `1.6` since the ones digit is a single character and `.` sorts before numeric strings.

Reply:
yeah, I knew that, this should be the easiest way to compare versions. With hardcoded versions we can simply compare strings.